### PR TITLE
Refactor qc

### PR DESCRIPTION
**Pull request by ypriverol**
@mwalzer some minor comments: 

- add new tests
- add files to new tests 
- minor update of the README 

@mwalzer the following file #qcplot.py contains a lot of R dependencies, i would like to remove it ;)

### CODE REVIEW

File: tests/test_basicqc.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import unittest
+import pyopenms as oms
+
+from qccalculator import basicqc, utils, idqc
+
+
+class MyTestCase(unittest.TestCase):
```
Comment (mwalzer) [inline tests/test_basicqc.py]: can you make the test class name a bit more descriptive? Otherwise all good.

**Review by mwalzer — CHANGES_REQUESTED** [review]
Can you make the test class name more descriptive? Everything else looks good.

File: tests/test_basicqc.py (hunk 1 above)
Comment (ypriverol) [reply tests/test_basicqc.py]: Done

**Comment by ypriverol** [comment]
@mwalzer changes has been included.

**Pull request merged**